Symfony/CS/Fixer/PSR2/ElseifFixer.php

```diff
@@ -30,25 +30,31 @@ class ElseifFixer extends AbstractFixer
     public function fix(\SplFileInfo $file, $content)
     {
         $tokens = Tokens::fromCode($content);
+        for ($index = 7, $count = count($tokens); $index < $count - 1; ++$index) {
```

Comment:
` foreach ($tokens as $index => $token) {`
is more readable than magic `7`, at least for me...